spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe User do
+  describe 'Associations' do
+    it { should have_many(:manifests) }
+  end
+
+  describe 'cdx_user_id' do
+    it 'is always required' do
+      user = create(:user)
+      expect(user.cdx_user_id).to_not be_nil    
```

Comment:
if that worked, it would be very cool.

here's what I tried and failed:
```diff
diff --git a/spec/models/user_spec.rb b/spec/models/user_spec.rb
index 4ae36a3..0e6b4ac 100644
--- a/spec/models/user_spec.rb
+++ b/spec/models/user_spec.rb
@@ -3,6 +3,8 @@ require 'rails_helper'
 describe User do
   describe 'Associations' do
     it { should have_many(:manifests) }
+    it { should validate_uniqueness_of(:cdx_user_id) }
+    it { should validate_presence_of(:cdx_user_id) }
   end
 
   describe 'cdx_user_id' do
```
```
User
  Associations
    should validate that :cdx_user_id is case-sensitively unique (FAILED - 1)
    example at ./spec/models/user_spec.rb:7 (FAILED - 2)
    should have many manifests
  cdx_user_id
    is always required
    is unique
    #find_or_create

Failures:

  1) User Associations should validate that :cdx_user_id is case-sensitively unique
     Failure/Error: it { should validate_uniqueness_of(:cdx_user_id) }
       User did not properly validate that :cdx_user_id is case-sensitively
       unique.
         The record you provided could not be created, as it failed with the
         following validation errors:
       
         * cdx_user_id: ["can't be blank"]
     # ./spec/models/user_spec.rb:6:in `block (3 levels) in <top (required)>'

  2) User Associations 
     Failure/Error: it { should validate_presence_of(:cdx_user_id) }
     NoMethodError:
       undefined method `validate_presence_of' for #<RSpec::ExampleGroups::User::Associations:0x007ff6ca35f138>
     # ./spec/models/user_spec.rb:7:in `block (3 levels) in <top (required)>'

Finished in 0.17316 seconds (files took 3.94 seconds to load)
6 examples, 2 failures
```